packages/js/product-editor/src/components/attribute-term-input-field/attribute-term-input-field.tsx

```diff
@@ -311,11 +311,7 @@ export const AttributeTermInputField: React.FC<
 												<CheckboxControl
 													onChange={ () => null }
 													checked={ isSelected }
-													label={
-														<span>
-															{ item.name }
-														</span>
-													}
+													label={ item.name }
```

Comment:
Have you checked if this causes a styling regression?

Reply:
Upon reviewing the type definition, it appears that CheckboxControl accepts a string for the label prop. However, the final consumer of this prop, BaseControl, expects the label to be of type ReactNode. I'm going to revert these changes.